### DISCUSSION TITLE
Is it possible to let only weblate push to my vcs?

### DISCUSSION THREAD

**Discussion by AwesomeStickz** [WeblateOrg/weblate#9474] [Q&A]
I'm trying to setup weblate in such a way that only weblate user pushes to my github repo instead of me + weblate, is that possible? I think adding as a collaborator _could_ work but it also would appear as an invitation so I'm not sure, is there any way to let it push commits without using my email?

**Comment by nijel** [comment]
Yes, just configure your component accordingly: https://docs.weblate.org/en/latest/admin/continuous.html#pushing-changes-from-weblate
